interop_binaries/src/bin/janus_interop_collector.rs

```diff
@@ -0,0 +1,503 @@
+use anyhow::Context;
```

Comment:
We should consider factoring some of this logic out into a standalone collector library that handles making collect requests, polling collect URIs and combining aggregate shares into aggregate results. We could distribute it in `janus_client`. Not needed in this PR, but a future enhancement.

Reply:
Agreed, but maybe it should go in a separate crate, to keep janus_client slim for WASM users.